src/styles/blocks/button.scss

```diff
@@ -0,0 +1,37 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 27px;
+  text-decoration: none;
+  color: $c-main-black;
```

Comment:
In my opinion, setting a fixed height, it's not the best practice. But in this case, maybe, it's allowed.